src/raw/README.md

```diff
@@ -0,0 +1,90 @@
+# cisco-sso/raw
+
+The `cisco-sso/raw` chart takes a list of raw Kubernetes resources and
+merges each resource with a default `metadata.labels` map and installs
+the result.
+
+Some use cases for this chart include Helm-based installation and
+maintenance of resources of kinds:
+- LimitRange
+- PriorityClass
+
+## Usage
+
+### STEP 1: Create a yaml file containing your raw resources.
+
+```
+# raw-priority-classes.yaml
+
+resources:
```

Comment:
I think we should just put the resources in the templates directory and name the chart `priority-classes` or something.  I don't personally like whats going on here.  It may grow on me, but I'm not there yet.

Reply:
Thanks @drekle.  I'll leave this open and see if others agree.

The general idea here is to use this raw chart for the nuts and bolts of Kubernetes that don't roll up into an application.  It's a lot of power to wield and surely can be abused.  Still, I believe it can be used for good.  Let's keep the dialog going.

Reply:
I'm fine with this model.  There are little bits and pieces that would be a mess if we created a chart for each of them.

Priority classes, default limits, storageclass controllers, etc.

Speaking of which, for storage classes, we often want to delete one, and install another.  How can we take care of that with a chart?